session_cleaner/README.md

```diff
@@ -36,14 +38,31 @@ script with the `--data-dir` argument to tell it where the Data directory is,
 so that the Sublime session file can be found. This can be a fully qualified
 path or a path relative to the current working directory.
 
+When you run the script, you must specify the items that you would like to have
+cleaned out of the session file by using the following command line arguments.
+Note that Sublime Merge only supports the first argument, but providing the
+others is harmless and will do nothing.
+
+  * `--workspaces` to clean up the list of project/workspace files (Sublime
+    Text) or git repositories (Sublime Merge) that no longer exist
+
+  * `--files` to clean up the list of recently accessed files that no longer
+    exist. (Note: If you had multiple windows open when you quit Sublime, this
+    action will trigger once for each window since they all carry their own
+    history).
```

Comment:
What does this "will trigger once for each window" part mean? Is this just fancy wording for "will clear unaccessable files for all windows that were open when ST was closed"?

Reply:
Yeah pretty much. The way the code loops through the bits of the session to clean it can be a little disconcerting to see what appears to be the same list of files scroll past multiple times.